src/Components/Todos/Todos.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  key: number;
+  onDelete: (id: number) => void;
+};
+
+export const Todos: React.FC<Props> = ({ todo, onDelete }) => {
```

Comment:
The file is named `Todos.tsx`, but the component represents todo. To follow the naming convention, please rename the component to `Todo` and update the file name accordingly to `Todo.tsx`. 📝